### PR TITLE
Add MSLogonII security type

### LINKED ISSUE #1197: Add support for UltraVNC MS-Logon II credentials
UltraVNC as an extended credentials/security type called [MS-Logon II](http://www.uvnc.com/features/authentication.html). It lets one use local or domain account credentials with an username and password instead of the tradicional VNC password.

This would be a nice addition to noVNC.

### PR DESCRIPTION
**Pull request by pdlan**
This PR adds UltraVNC's MSLogonII security type, which resolves #1197.

### CODE REVIEW

File: core/rfb.js [hunk 1]
```diff
@@ -62,6 +62,7 @@ const securityTypeTight             = 16;
 const securityTypeVeNCrypt          = 19;
 const securityTypeXVP               = 22;
 const securityTypeARD               = 30;
+const securityTypeUltraVNCMSLogonII = 113;
```
Comment (CendioOssman) [inline core/rfb.js:65]: Perhaps just "MSLogonII" is sufficient? Keeps the length down on identifiers.

File: tests/test.rfb.js [hunk 2]
```diff
@@ -2,7 +2,6 @@ const expect = chai.expect;
 
 import RFB from '../core/rfb.js';
 import Websock from '../core/websock.js';
-import DES from "../core/des.js";
```
Comment (CendioOssman) [inline tests/test.rfb.js]: Please merge this in to the previous commit.

**Review by CendioOssman — CHANGES_REQUESTED** [review]
Nice, thank you! Looks good, with just two small comments.

**Review by CendioOssman — APPROVED** [review]
Nice. Thank you for your contribution!

**Comment by Neustradamus** [comment]
@pdlan: Thanks a lot!

**Pull request merged**